tracer/src/Datadog.Trace/AppSec/Waf/Result.cs

```diff
@@ -4,6 +4,7 @@
 // </copyright>
 
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
Perhaps consider adding `#nullable enable` to this file too? 🤔

Reply:
added in a5602d6a758312da7234698a1997fedd25879199